man/systemd.exec.xml

```diff
@@ -972,6 +972,13 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         configuration or lifetime guarantees, please consider using
         <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
 
+        <para>These options are not designed to be changed to use a different prefix by distribution packaging of software,
+        for example by changing the <varname>@localstatedir@</varname> autoconf variable from its default value of
+        <filename>/var</filename>. If software needs to support installation in a different prefix, it is recommended
+        that systemd support be disabled for that build of the software. It is not recommended to routinely use
+        <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry> instead
+        of these options, as it needs to be run manually to ensure the directories are created before the unit is started.</para>
```

Comment:
I'll admit that that I head to read the ticket before I could understand what this para is about. In particular, at first blush, it'd think that the mentions of `@localstatedir@` here refer to systemd. Also, I don't think we should talk about autoconf and autoconf-specific details in systemd docs. Maybe just say:
"""
The directories defined by these options are always created under the standard paths used by systemd (/var, /run, /etc, ...). If the service needs directories in a different location, a different mechanism has to be used to create them.

(separate para) tmpfiles.d(5) provides functionality that overlaps with these options. Using these options is recommended, because the lifetime of the directories is tied directly to the lifetime of the unit, and it is not necessary to ensure that the tmpfiles.d configuration is executed before the unit is started.
"""

Reply:
Sounds good to me, thanks for reworking it. I’ve updated the patch with a formatted version of your suggestion.